File: app/src/main/java/trunk/doi/base/ui/fragment/info/mvp/InfoPresenter.java
package trunk.doi.base.ui.fragment.info.mvp;


import com.base.lib.di.scope.FragmentScope;
import com.base.lib.mvp.BasePresenter;

import javax.inject.Inject;


@FragmentScope
public class InfoPresenter extends BasePresenter<InfoContract.Model, InfoContract.View> {

    @Inject
    public InfoPresenter(InfoContract.Model model, InfoContract.View rootView) {
        super(model, rootView);
    }


    @Override
    public void onDestroy() {
        super.onDestroy();

    }
}

File: app/src/main/java/com/gas/app/ui/fragment/info/mvp/InfoPresenter.kt
package com.gas.app.ui.fragment.info.mvp

import com.base.lib.di.scope.FragmentScope
import com.base.lib.mvp.BasePresenter
import javax.inject.Inject

@FragmentScope
class InfoPresenter @Inject constructor(model: InfoContract.Model, rootView: InfoContract.View) : BasePresenter<InfoContract.Model, InfoContract.View>(model, rootView) {
    override fun onDestroy() {
        super.onDestroy()
    }
}